test/framework/cvo.go

```diff
@@ -70,6 +70,15 @@ func DisableCVOForOperator(te TestEnv) {
 	})
 	changed = changed || componentChanged
 
+	cv.Spec.Overrides, componentChanged = addCompomentOverride(cv.Spec.Overrides, configv1.ComponentOverride{
```

Comment:
Would be nice with a comment here providing some background on why we're disabling MCO.
Wdyt?